ApplicationLibCode/FileInterface/RifFaultReactivationModelExporter.cpp

```diff
@@ -529,6 +531,37 @@ std::pair<bool, std::string>
         RifInpExportTools::printHeading( stream, "INCLUDE, input=" + fileName );
     }
 
+    if ( stressFromGrid )
+    {
+        std::string stressName = "STRESS";
+
+        // Export the stress to a separate inp file for each step
+        std::string fileName = stressName + ".inp";
+        std::string filePath = createFilePath( exportDirectory, fileName );
+
```

Comment:
Should step name and model name be included in file name?

Reply:
I think maybe not since it is always the first time step?

Reply:
Comment seemed to indicate it could be multiple steps, but the model name should probably be included in case multiple models are stored in the same folder

Reply:
Yes, the comment is wrong. Will Fix.